spec/features/user_can_view_admin_dashboard_spec.rb

```diff
@@ -0,0 +1,28 @@
+require "rails_helper"
+
+feature "admin dashboard" do
+
+  it "a guest can not view the admin dashboard" do
+    visit '/admin/dashboard'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.